modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -3544,6 +3544,110 @@ RETURN *
 ======
 =====
 
+[#_neo_clientnotification_statement_redundantoptionalprocedure]
+=== Redundant Optional Procedure
+
+.notification details
+[cols="<1s,<4"]
+|===
+|neo4j code
+m|neo.clientnotification.statement.redundantoptionalprocedure
+|title
+a|The use of `OPTIONAL` is redundant when the procedure calls a void procedure.
+|description
+|The use of `OPTIONAL` is redundant as `CALL %s` is a void procedure.
+|category
+m|generic
+|gqlstatus code
+m|03n61
+|status description
+a|info: redundant optional procedure. The use of `OPTIONAL` is redundant as `CALL %s` is a void procedure.
+|classification
+m|generic
+|severitylevel
+m|information
+|===
+
+.Redundant use of optional on a procedure call.
+[.tabbed-example]
+=====
+[.include-with-neo4j-code]
+======
+Query::
++
+[source,cypher]
+----
+OPTIONAL CALL db.createLabel("A")
+----
+
+Description of the returned code::
+Optional is redundant in the case of void procedures. The use of `OPTIONAL` on void procedures have no effect and can be removed.
+
+Suggestions for improvement::
+If the intended behavior of the query is to use a void procedure the `OPTIONAL` keyword can be removed without impacting the query.
++
+[source,cypher]
+----
+CALL db.createLabel("A")
+----
+======
+=====
+
+[#_neo_clientnotification_statement_redundantoptionalsubquery]
+=== Redundant Optional Subquery
+
+.notification details
+[cols="<1s,<4"]
+|===
+|neo4j code
+m|neo.clientnotification.statement.redundantoptionalsubquery
+|title
+a|The use of `OPTIONAL` is redundant when `CALL` is a unit subquery.
+|description
+|The use of `OPTIONAL` is redundant as `CALL` is a unit subquery.
+|category
+m|generic
+|gqlstatus code
+m|03n62
+|status description
+a|info: redundant optional subquery. The use of `OPTIONAL` is redundant as `CALL` is a unit subquery.
+|classification
+m|generic
+|severitylevel
+m|information
+|===
+
+.Redundant use of optional on a call subquery.
+[.tabbed-example]
```

Comment:
This one also needs a tab example for the GQLSTATUS code, similar to the rest of the notifications.

Reply:
I think I've misunderstood the format. I'm pushing some changes to align with the rest of the notifications. I'll make sure that the notifications in the monorepo correspond to what is written here, in substance and formatting.